apps/portals/stopadportal/src/config/style/_style_overrides.scss

```diff
@@ -30,7 +31,7 @@ footer {
 }
 
 #home-link {
-  @media (max-width: map-get(SrcVariables.$breakpoints, 'medium')) {
+  @media (max-width: map.get(SrcVariables.$breakpoints, 'medium')) {
```

Comment:
Have to replace builtin functions with module versions: https://css-tricks.com/introducing-sass-modules/#aa-import-built-in-sass-modules